src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -249,22 +250,29 @@
             }
          }
       },
-	  {
+      {
        "Storage back-end Settings":{
             "unity_storage_protocol" : {
                   "valid_values": [
                   "iSCSI",
                   "FC"
+            }
+       },
+      {
+         "PowerFlex Settings": {
+            "enable_powerflex_backend": {
+               "valid_values": [
+                  "true",
+                  "false"
                ]
-        },
-             "powermax_protocol" : {
-                  "valid_values": [
-                  "iSCSI",
-                  "FC"
+            },
+            "powerflex_rpms_method": {
+               "valid_values": [
+                  "local",
+                  "external"
                ]
-        }
-
-       }
+<<<<<<< HEAD
```

Comment:
Have to resolve this

Reply:
Don't know why it's there. Supposed to got it resolved.